src/viur/core/skeleton.py

```diff
@@ -1442,6 +1501,35 @@ def __init__(self, baseSkel=None):
         self.customQueryInfo = {}
 
 
+# Module functions
+
+
+def skeletonByKind(kindName: str) -> t.Type[Skeleton]:
+    """
+        Returns the Skeleton-Class for the given kindName. That skeleton must exist, otherwise an exception is raised.
+        :param kindName: The kindname to retreive the skeleton for
+        :return: The skeleton-class for that kind
```

Comment:
Please unindent these docstrings according to our [Coding Convention](https://github.com/viur-framework/viur-core/blob/main/CONTRIBUTING.md#coding-convention), where we follow [PEP 257](https://peps.python.org/pep-0257). :)

```suggestion
    Returns the Skeleton-Class for the given kindName. That skeleton must exist, otherwise an exception is raised.
    :param kindName: The kindname to retreive the skeleton for
    :return: The skeleton-class for that kind
```